File: Python-Programing-Basics/Conditional Statments/Conditional Statements - Exercises/02_bonus_score/bonus_score.py
def calculate_bonus_points1(score):
    bonus_points = 0
    if score % 2 == 0:
        bonus_points += 1
    elif score % 10 == 5:
        bonus_points += 2
    bonus_points = calculate_bonus_points2(score, bonus_points)

    print(bonus_points)
    print(score + bonus_points)


def calculate_bonus_points2(score, b_points):
    if score <= 100:
        b_points += 5
    elif score <= 1000:
        b_points += score * 0.20
    elif score > 1000:
        b_points += score * 0.10

    return b_points


points = int(input())
calculate_bonus_points1(points)

File: JavaScript/Programming Basics - JavaScript/Conditional Statements/bonusScore.js
function getBonusPoints (input) {
    let points = Number(input[0]);
    let bonusPoints = 0;

    if (points <= 100) {
        bonusPoints += 5;
    } else if (points > 1000) {
        bonusPoints = points * 0.10;
    } else {
        bonusPoints = points * 0.20
    }
    
    if (points % 2 === 0) {
        bonusPoints += 1;
    }
    if (points % 10 === 5) {
        bonusPoints += 2;
    }

    console.log(bonusPoints);
    console.log(points + bonusPoints);
}

getBonusPoints(["20"]);
getBonusPoints(["175"]);
getBonusPoints(["2703"]);
getBonusPoints(["15875"]);